### PR TITLE
Clement - JS - week5

### PR DESCRIPTION
**Pull request by clems27**
Dom- manipulation

### CODE REVIEW

File: week-5-practise/js/main.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+var blueButton = document.querySelector("#blueBtn");
+var jumbotron = document.querySelector(".jumbotron");
+var donateAndVolunteerButtons = document.querySelectorAll(".btn-lrg");
+blueButton.addEventListener("click", donateAndVolunteersBlueButton);
+
+//this function will change the  jumbotron class background color  and the class
+//button background / style colors
+
+function donateAndVolunteersBlueButton() {
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: Can you think of a more meaningful name for this function? Usually you'd go for a verb or an action that describes what the function does.

File: week-5-practise/js/main.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+var nameTextField = document.querySelector("#example-text-input");
+var emailTextField = document.querySelector("#exampleInputEmail1");
+var describeYourselfTextField = document.querySelector("#exampleTextarea");
+
+//This function is to check if the elements are valid or not
+function areValidElements() {
+  let isvalid = true;
+  if (nameTextField.value.length < 1) {
+    nameTextField.style.backgroundColor = "red";
+    isvalid = false;
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: It's good practice to start boolean variable names on `is` or `has` – good work! Just a detail: Make sure to consistently use camel case (`isValid`).

File: week-5-practise/js/main.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+  }
+  if (
+    emailTextField.value.indexOf("@") !== -1 ||
+    emailTextField.value.length < 1
+  ) {
+    emailTextField.style.backgroundColor = "red";
+    isvalid = false;
+  }
```
Comment (foaly-nr1) [inline week-5-practise/js/main.js]: The only thing missing is changing the red background colour back to normal once the fields have been filled in correctly.

File: week-5/A-dom-manipulation/exercise.js [hunk 4]
```diff
@@ -14,20 +17,38 @@ Task 2
...
 
 When the 'Larger links!' button is clicked, the text of all links on the page should increase.
-*/
\ No newline at end of file
+*/
+var largeButtonText = document.querySelector("#largerLinksBtn");
+largeButtonText.addEventListener("click", largeBTextIncrease);
+function largeBTextIncrease() {
+    document.querySelectorAll("a").forEach(function (largeButtonText) {
+        largeButtonText.style.fontSize = "3em"
```
Comment (foaly-nr1) [inline week-5/A-dom-manipulation/exercise.js]: Nice use of `forEach()` and responsive font size units. 🙂

File: week-5-practise/js/main.js (hunk 1 above)
Comment (clems27) [reply week-5-practise/js/main.js]: @foaly-nr1  thanks for sharing the above, I find these part to be tricky sometimes, Will rename.

File: week-5-practise/js/main.js (hunk 2 above)
Comment (clems27) [reply week-5-practise/js/main.js]: Thanks for the remainder, Hope to Improve on it next time

File: week-5-practise/js/main.js (hunk 3 above)
Comment (clems27) [reply week-5-practise/js/main.js]: Thanks will make correction

File: week-5/A-dom-manipulation/exercise.js (hunk 4 above)
Comment (clems27) [reply week-5/A-dom-manipulation/exercise.js]: Thanks

**Pull request closed without merging**